ruby/vampires.rb

```diff
@@ -1,4 +1,4 @@
-puts "How many employees will be processed (please enter a number)?"
+puts "How many employees will be processed? (please enter a number)"
 employees = gets.chomp.to_i
 	number = 0
 	until number == employees
```

Comment:
Make sure that you are indenting everything nested under your until for readability.  I use https://github.com/airbnb/ruby as a style guide when Im wondering how to format my code.